src/ui/Forms/Main.cs

```diff
@@ -9486,8 +9486,8 @@ private void TextBoxListViewTextKeyDown(object sender, KeyEventArgs e)
             {
                 if (_subtitle.Paragraphs.Count > 0 && SubtitleListview1.SelectedItems.Count >= 1 && SubtitleListview1.SelectedItems.Count < 10)
                 {
-                    e.SuppressKeyPress = true;
                     SplitSelectedLineBilingual();
+                    e.SuppressKeyPress = true;
```

Comment:
Don't you think this looks better switched? Like all the others, `e.SuppressKeyPress = true;` should be at the end.